MapboxNavigationUI/RouteMapViewController.swift

```diff
@@ -0,0 +1,349 @@
+import UIKit
+import Mapbox
+import MapboxDirections
+import Pulley
+import MapboxNavigation
+import SDWebImage
+
+
+class ArrowFillPolyline: MGLPolylineFeature {}
+class ArrowStrokePolyline: ArrowFillPolyline {}
+
+
+class RouteMapViewController: UIViewController, PulleyPrimaryContentControllerDelegate {
+    @IBOutlet weak var mapView: MGLMapView!
```

Comment:
@bsudekum was concerned that the application wouldn’t have access to this MGLMapView for the purposes of customization, such as specifying a style other than Streets v10 (which the example projects use in their main maps) or Streets v9 (which this MGLMapView uses).